packages/valory/skills/market_creation_manager_abci/rounds.py

```diff
@@ -333,6 +336,9 @@ def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Event]]:
             if self.most_voted_payload == self.REDEEM_BOND_DONE_PAYLOAD:
                 return self.synchronized_data, Event.REDEEM_BOND_DONE
 
+            if self.most_voted_payload == self.DEPOSIT_DAI_DONE_PAYLOAD:
+                return self.synchronized_data, Event.DEPOSIT_DAI_DONE
```

Comment:
`DEPOSIT_DAI_DONE` missing from `PostTransactionRound`.

Reply:
You mean in transition function, right?
Fixed